doc/vic_installation/pre_post_install_scripts.md

```diff
@@ -0,0 +1,74 @@
+
+- Pre-initialization scripts run immediately after the mount of the Docker metadata disk, before the initialization and setup of the appliance.
+- Post-initialization scripts run after the initialization and setup of the appliance, but before the Docker daemon starts.
+
+**NOTE** Any pre- or post-initialization configuration that you perform via scripting is unsupported. This functionality is provided purely so that you can test the vSphere Integrated Containers technical preview in your specific vSphere environment.
+
+For example, you might require the vSphere Integrated Containers appliance to have an additional network interface with a static IP address, so that you can expose the virtual container host to users who do not have access to the vCenter Server management network. This is not possible with the current installer options, which do not allow for multiple network interfaces. Achieving such a configuration requires you to reconfigure the vSphere Integrated Containers appliance after deployment. 
+
+Similarly, your vSphere environment might also require TLS authentication to be persistent across reboots of the vSphere Integrated Containers appliance.   
+
+**Prequisites**
+
+- Deploy the vSphere Integrated Containers appliance.
+
+ To use pre- and post-initialization scripts, you must use build 58 or later of the vSphere Integrated Containers command line installer.
+- Verify that the deployment was successful and obtain the address of the appliance.
+
+**Procedure**
+
+1. Use SSH to log in to the vSphere Integrated Containers appliance as `root`.
+3. Create folders named `pre`, or `post`, or both, under `/var/lib/docker/`.
+4. Copy script files into the `pre` or `post` folders, or create the scripts directly in those folders.
+
+ - Place pre-initialization scripts in the `pre` folder.
+ - Place post-initialization scripts in the `post` folder.
+ - Use `#!/bin/ash` as the script interpreter
+ - Ensure that the scripts are executable.
+6. Restart the vSphere Integrated Containers appliance.
+
+**Example: Implementing Persistent TLS Authentication**
+
+To implement TLS authentication that is persistent across reboots of the vSphere Integrated Containers appliance, you must provide a post-initialization script to create the certificates and provide them to the Docker daemon. 
+
+1. Use SSH to log in to the vSphere Integrated Containers appliance as `root`.
+2. Create a script file named `cert.sh`.
+3. Copy and paste the following script into `cert.sh`.
+
+ <pre>#! /bin/ash
```

Comment:
Honestly this whole script may be a little too detailed. We could include it in an appendix perhaps.
It may be simpler to point people at this page: https://docs.docker.com/engine/security/https/